libs/cardano-ledger-core/src/Cardano/Ledger/Credential.hs

```diff
@@ -96,20 +96,20 @@ instance NoThunks (Credential kr c)
 instance Crypto c => ToJSON (Credential kr c) where
   toJSON (ScriptHashObj hash) =
     Aeson.object
-      [ "script hash" .= hash
+      [ "scripthash" .= hash
```

Comment:
This will make the existing genesis file for shelley (at least) incompatible, wont' that be a problem?

Reply:
Why do you think that? There aren't any credentials in ShelleyGenesis.
https://github.com/input-output-hk/cardano-node/blob/master/configuration/cardano/mainnet-shelley-genesis.json

Reply:
Ah interesting.  In the genesis file in our repo there is a "staking" object with "stake" and "pools":  https://github.com/input-output-hk/cardano-ledger/blob/master/eras/shelley/test-suite/test/Golden/ShelleyGenesis#L44

They are coming from `ShelleyGenesisStaking`: https://github.com/input-output-hk/cardano-ledger/blob/master/eras/shelley/impl/src/Cardano/Ledger/Shelley/Genesis.hs#L115

I just checked the config in sanchonet and both of these appear empty: 
```
 "staking": {
        "pools": {},
        "stake": {}
    },
```
I guess the cardano-node repo is the most relevant source.

Reply:
Aha. That's because those two fields are hacks :smile: they aren't really part of real ShelleyGenesis. They are used for testing and benchmarking and I am working on extracting those two fields (`initialFunds` and `staking`) out of ShelleyGenesis

Reply:
Ah I see. Good to know, thanks for the explanation :+1: